src/applications/accredited-representative-portal/components/POARequestsCard/POARequestsCard.jsx

```diff
@@ -0,0 +1,137 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+export const createLimitationsCell = (
+  isTreatmentDisclosureAuthorized,
+  isAddressChangingAuthorized,
+) => {
+  const limitations = [];
+
+  // If do not authorize sharing health info or authorize change of address then we label it as a limitation of consent
+  if (!isTreatmentDisclosureAuthorized) limitations.push('Health');
+  if (!isAddressChangingAuthorized) limitations.push('Address');
+
+  return limitations.length > 0 ? limitations.join(', ') : 'None';
+};
+
+export const formatDate = date => {
+  const [month, day, year] = new Date(date)
+    .toLocaleDateString('en-US', {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+    })
+    .split(' ');
+  return `${month} ${day} ${year}`;
+};
+
+export const checkDaysLeft = date => {
```

Comment:
Refactor tip:
```javascript
import { differenceInDays } from 'date-fns';

const EXPIRES_SOON_THRESHOLD_DAYS = 7;

const expiresSoon = poaRequest => {
  const now = new Date();
  const expiresAt = new Date(poaRequest.expiresAt);
  return differenceInDays(expiresAt, now) < EXPIRES_SOON_THRESHOLD_DAYS;
}
```
Then, below:
```jsx
{expiresSoon(poaRequest) && <va-icon
  class="poa-request__card-icon"
  icon="warning"
  size=2
  srtext="warning"
  aria-hidden="true"
/>}

Reply:
Given this new [change to the design](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97453#issuecomment-2504695700) to include a duration, we should leverage platform's existing utility function [`timeFromNow`](https://github.com/department-of-veterans-affairs/vets-website/blob/cec0e38b394778a1830b4030aca6f4b4e600946d/src/platform/utilities/date/index.js#L115):
> ...we included a line after the date shown that counts down the days to expiration. I.e: "POA request expires on: November 25, 2024 (in 5 days)"

I think we should still use the daylight-savings-time-insensitive version of `expiresSoon` for our predicate function, even though it will have a slight discrepancy during daylight savings time. This is because it is so simple and flexible.

Reply:
@nihil2501 I am using the existing `timeFromNow` function in my last update

Reply:
ok, I saw the slack chat earlier about the badge height discrepancy but not the new text requirement. updated and pushed up